pkg/agent/proxy/proxier.go

```diff
@@ -173,15 +174,13 @@ func (p *proxier) removeStaleServices() {
 				continue
 			}
 		}
-		// Remove Service group whose Endpoints are local.
-		if svcInfo.ExternalPolicyLocal() {
-			if groupIDLocal, exist := p.groupCounter.Get(svcPortName, true); exist {
-				if err := p.ofClient.UninstallServiceGroup(groupIDLocal); err != nil {
-					klog.ErrorS(err, "Failed to remove Group of local Endpoints for Service", "Service", svcPortName)
-					continue
-				}
-				p.groupCounter.Recycle(svcPortName, true)
+		// Remove Service group which has only local Endpoints.
```

Comment:
the `svcInfo.ExternalPolicyLocal()` condition was removed, yet the commend still mentions "local Endpoints". Is that correct?

Reply:
> So there was a bug regarding this? a group may never be deleted?

For a ClusterIP, when `svcInfo.InternalPolicyLocal()` is true, a group may never be delete.